### DISCUSSION TITLE
Dịch hay giữ "cell", "markdown", "notebook"

### DISCUSSION THREAD

**Discussion by rootonchair** [d2l-ai/d2l-vi#2987] [Terminology]
Đây là các từ khá mới và chắc chắn là chưa có phần dịch tiếng Việt. Các từ này xuất hiện rất thường xuyên trong chương jupyter. Hiện tại mình đang dịch "cell" -> "ô", "markdown" -> "đánh dấu", "notebook" -> "sổ tay"

Mọi người cho ý kiến nhé

**Comment by lkhphuc** [comment]
Cell mấy thấy cứ dịch là `ô` cũng được, còn `markdown` và `notebook` thì không nên.

`Markup` mới là `đánh dấu`, còn `markdown` là chơi chữ của markup thôi, bây giờ nó là tên một định dạng rồi.
`Notebook` thì đúng là mượn từ nghĩa `sổ ghi chép` thật, nhưng mình thấy bây giờ nó gần như cũng thành tên một định dạng giống markdown rồi nên không ủng hộ việc dịch.

**Reply by rootonchair** [reply]
Vậy em sẽ chốt theo ý của anh @lkhphuc

**Comment by rootonchair** [comment]
@minhduc0711 @trGiang99 @davidnvq @duythanhvn @cuongvng

**Reply by thanhcsf** [reply]
Anh cùng ý kiến với Phúc ở điểm này nha em.
